qcodes/dataset/sqlite/db_upgrades/__init__.py

```diff
@@ -254,8 +254,8 @@ def perform_db_upgrade_6_to_7(conn: ConnectionPlus) -> None:
     """
     Perform the upgrade from version 6 to version 7
 
-    Add a captured_runid and captured_counter column to the runs table and
-    assign the value from the run_id and counter to these columns.
+    Add a captured_run_id and captured_counter column to the runs table and
+    assign the value from the run_id and result_counter to these columns.
```

Comment:
This should indeed really be called `result_counter` but its called counter internally :( not sure what is best